phoenix-core/src/main/java/org/apache/phoenix/compile/ProjectionCompiler.java

```diff
@@ -507,6 +507,10 @@ public static RowProjector compile(StatementContext context, SelectStatement sta
                 } else {
                     for (byte[] cq : entry.getValue()) {
                             PColumn column = family.getPColumnForColumnQualifier(cq);
+                            // Continue: If an EMPTY_COLUMN is in the projection list,
+                            // since the table column list does not contain the EMPTY_COLUMN
+                            // no value is returned.
+                            if (column == null) continue;
```

Comment:
nit: Can we use braces and put the `continue` on a separate line to be consistent with formatting?